habitat-lab/habitat/datasets/rearrange/run_episode_generator.py

```diff
@@ -50,6 +50,8 @@ class RearrangeEpisodeGeneratorConfig:
     additional_object_paths: List[str] = field(
         default_factory=lambda: ["data/objects/ycb/"]
     )
+    # optionally correct unstable states by removing extra unstable objects (within minimum samples limitations)
+    correct_unstable_results: bool = False
```

Comment:
Optional feature because:
1.  backwards compatibility
2. user can choose if culling is allowed for their purposes (default True would be ok for this)
3. Doesn't work with target sampling yet

Reply:
Can you add a comment explaining just that?